internal/executor/vaultunboxer/jwtauth.go

```diff
@@ -19,5 +22,9 @@ func (jwtAuth *JWTAuth) Login(ctx context.Context, client *vault.Client) (*vault
 		data["role"] = jwtAuth.Role
 	}
 
-	return client.Logical().WriteWithContext(ctx, "auth/jwt/login", data)
+	if jwtAuth.Path == "" {
+		jwtAuth.Path = "jwt"
```

Comment:
@fkorotkov I'd rather prefer to reduce the power of this variable to only specifying a single path component (or more correctly, authentication method mount point), because otherwise this abstraction will start to leak once we'll begin supporting other authentication methods, which have wildly different base URLs, for example:

<img width="1362" alt="Screenshot 2022-10-26 at 18 03 01" src="https://user-images.githubusercontent.com/85709/198047312-7b280de7-c3ac-4126-a90a-14bb7fadff2e.png">

Moreover, `VAULT_AUTH_PATH` seems to be already a non-spoken standard, judging from the [GitLab's documentation](https://docs.gitlab.com/ee/ci/secrets/) (search for `VAULT_AUTH_PATH`).

Reply:
Agreed. Thanks for the clarification.